pages/agent/v3/docker.md.erb

```diff
@@ -15,9 +15,9 @@ You can run the Buildkite Agent inside a Docker container using the [`buildkite/
 
 ## 🚨 Incompatibilities with plugins
 
-Whilst the agnt runs well in docker, there are presently some difficulties when combining your agent in a docker container and using many of our [plugins](https://buildkite.com/plugins). These can largely be mitigated by ensuring that the `builds-path` from the host is bind mounted into your agent container with the same path as on the host and having the buildkite-agent installed on the host so that it can be mounted into containers.
+While the agent runs well in Docker, there are presently some difficulties when running an agent in a Docker container and using many of our [plugins](https://buildkite.com/plugins). These issues can largely be mitigated by ensuring that the `builds-path` from the host is bind mounted into your agent container with the same path as on the host, and having the `buildkite-agent` installed on the host so that it can be mounted into containers.
```

Comment:
I feel like the last part of the paragraph could do with an explicit example of how to run the agent in a plugin-compatible way. For example (super rough and unchecked), a common configuration is:

```shell
docker run \
  # Make sure the build checkout dir exists on the host machine, not just the agent container
  -v /var/lib/buildkite/builds:/var/lib/buildkite/builds \
  # Make sure we use the same path in the container
  -e “BUILDKITE_BUILDS_PATH=/var/lib/buildkite/builds” \
  # Some Buildkite plugins, like the Docker plugin, mount the buildkite-agent binary into the job’s container to do artifact and meta-data
  -v “$(which buildkite-agent):/usr/local/bin/buildkite-agent” \
  # Let the agent run Docker commands, so people can use the Docker and Docker Compose Buildkite plugins
  -v “/var/run/docker.sock:/var/run/docker.sock” \
  buildkite/agent:3 start
```

People can then figure out how to do something like that using ECS/k8s etc if they need.

Should we also add a caveat that they should understand what container security protections they’re bypassing by setting these particular options?

Reply:
Wow I really made a mess of english in that line huh. 

I think your suggestion makes a heap of sense @toolmantim, that's where I was hoping to end up but wanted to chat about it a bit first.